boards/common/esp32c3/include/board_common_esp32c3.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2025 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
```

Comment:
```suggestion
 * directory for more details.
 */

#pragma once

/**
```

We recently started to migrate to `#pragma once` instead of classic include guards.